bundles/org.openhab.core.config.core/src/test/java/org/openhab/core/config/core/ConfigurationTest.java

```diff
@@ -62,6 +66,7 @@ public void assertGetConfigAsWorks() {
         configuration.put("stringField", "test");
         configuration.put("enumField", "ON");
         configuration.put("listField", List.of("one", "two", "three"));
+        configuration.put("setField", List.of("one", "two", "three"));
```

Comment:
Shouldn't thi be as `Set`?

```suggestion
        configuration.put("setField", Set.of("one", "two", "three"));
```

Reply:
No, I deliberately chose to use List as input to check if a List is properly converted to a Set if that is required. Maybe another Set->Set and Set->List test should be added.